Capfile

```diff
@@ -15,15 +15,14 @@ require 'capistrano/deploy'
 #   https://github.com/capistrano/rails
 #
 # require 'capistrano/rvm'
-# require 'capistrano/rbenv'
+require 'capistrano/rbenv'
 # require 'capistrano/chruby'
 require 'capistrano/bundler'
 require "capistrano/scm/git"
 install_plugin Capistrano::SCM::Git
 #require 'capistrano/rails/assets'
 #require 'capistrano/rails/migrations'
 require 'capistrano/rails'
-require 'capistrano/yarn'
```

Comment:
removing `capistrano/yarn', was required?

Reply:
Not totally sure, but removing `capistrano/yarn` may cause an issue for AgroPortal because they're using [jsbundling-rails](https://github.com/rails/jsbundling-rails), which requires yarn to be installed. BioPortal is headed in the same direction.

Reply:
ok, then we should not remove yarn then

Reply:
Yes, that was my concern.